__tests__/Snapshottest/__snapshots__/UIOutputComparisonTests.tsx.snap

```diff
@@ -75,13 +75,13 @@ exports[`LibraryContainer Test UI rendering of Library Item and child components
       "on": [Function],
       "raiseEvent": [Function],
       "reactor": Reactor {
-        "events": Array [],
+        "events": [],
```

Comment:
Was this part of this update? Why?

Reply:
When I ran the tests there was a snapshot mismatch because of changes made in the past so I've ran the snapshot again.